blackmarket/Managers/ViewRouter.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  ViewRouter.swift
+//  blackmarket
+//
+//  Created by Daniel Parra on 12/6/22.
+//
+
+import SwiftUI
+
+class ViewRouter: ObservableObject {
+    @Published var currentPage: Page = .home
+}
+
+enum Page {
```

Comment:
Maybe we can rename this to TabPage or similar, to make clear where it is intended to be used